sdks/go/test/integration/xlang/expansion_test.go

```diff
@@ -65,7 +47,11 @@ func TestAutomatedExpansionService(t *testing.T) {
 		t.Errorf("failed to start expansion service JAR, got %v", err)
 	}
 
-	checkPort(t, expansionPort, 15*time.Second)
+	conn, err := grpc.Dial("localhost:"+expansionPort, grpc.WithInsecure(), grpc.WithTimeout(15*time.Second))
```

Comment:
Need to add grpc.WithBlock(), since the default Dial is non-blocking.

Reply:
Fixed + moved to DialContext()